arch/arm64/boot/dts/qcom/sdm625-xiaomi-daisy.dts

```diff
@@ -190,6 +221,54 @@
 	};
 };
 
+&i2c_2 {
+	status = "okay";
+	speaker_codec: spkamp@3a {/* max98927 smartpa device*/
```

Comment:
maybe `audio-codec` is a better node name than `spkamp`?
Also I think the comment at the end of the line is somewhat pointless because the exact model is in the compatible anyways